gm4_balloon_animals/data/gm4_balloon_animals/functions/wandering_trader/pick_two_animals.mcfunction

```diff
@@ -0,0 +1,17 @@
+# Adds animal trades to the trader's trade pool.
+# @s = a wandering trader without the gm4_calling_bell_trader tag (aka naturally spawned trader) with exactly 2 trader llamas
+# at @s
+# from gm4_balloon_animals:wandering_trader/init_trader
+
+# calculate numbers based on summoned marker's UUID[0] and [3]
+execute summon marker run function gm4_balloon_animals:wandering_trader/animal_one/random_calc
+execute summon marker run function gm4_balloon_animals:wandering_trader/animal_two/random_calc
```

Comment:
Renaming the animals to `primary_animal` and `secondary_animal` would also make it so that the naming is consistent between function names and in-code (vs your current naming scheme which uses `animal1`, `animal_one`, `Animal 1`, and `a1_...`).

Reply:
Following up here on your previous question about the config.  So within the second one, and what would be any additional, is a check that the type does not match any existing mob types for this trader's animal offerings.

This solution was decided upon after a conversation with misode to prevent duplicate names, where every name is associated with a specific animal.  Each loot table roll cannot check against the other rolled types or the more concerning part, names.

So as of right now, there is no "config", the number is commented out and instead has a fixed 2.   Otherwise I would have written more of this in Python (before I knew about Jinja😍)

Reply:
~don't let special see what you wrote there~
Alright, let's stick to misode's word then.

Tangentially realted: Is the plan to support expansions and let other data packs sell their custom mobs?

Reply:
The long-term goal, yes!  I'm happy to have expansions, but loot tables are a bit of a pain to expand upon rn, right?  If there's a way I'm missing, I can implement that to push immediate expandability